pxtpy/converter.ts

```diff
@@ -491,7 +497,7 @@ namespace pxt.py {
         error(a, 9500, U.lf("types not compatible: {0} and {1}", t2s(t0), t2s(t1)))
     }
 
-    function typeCtor(t: Type): any {
+    function typeCtor(t: Type): string | SymbolInfo | {} | null {
```

Comment:
Does this type do anything? Wouldn't it evaluate to just == `{} | null`?

Reply:
I'm not sure what you mean. All four types can be returned.

Reply:
String and SymbolInfo are both subtypes of {}, but doesn't really make a difference - I suppose this will probably show up with the other two possible types explicitly in intellisense / as hover text, which is better.